examples/plot_3_capture_repr.py

```diff
@@ -5,10 +5,10 @@
 
 This example demonstrates how the configuration ``capture_repr``
 (:ref:`capture_repr`) works. The default ``capture_repr`` setting is
-``capture_repr: ('_repr_html_', '__repr__')`` and this was used to build this
-documentation. The output that is captured with this setting can be observed in
-this example. Differences in outputs that would be captured with other
-``capture_repr`` settings is also explained.
+``capture_repr: ('_repr_html_', '__repr__')`` and was used when building the
+Sphinx-Gallery documentation. The output that is captured with this setting
+is demonstrated in this example. Differences in outputs that would be captured
+with other ``capture_repr`` settings is also explained.
 """
 #%%
 # Nothing is captured for the code block below because no data is directed to
```

Comment:
I would end this sentence by saying "the last statement is an assignment, not an expression."